clarifiy support for relative paths in additional contexts (#339)
compose-spec/compose-spec: build.md

    source: https://github.com/myuser/project.git
```

> When used as a list, the syntax should follow the `NAME=VALUE` format, where `VALUE` is a string. Validation beyond that is the responsibility of the image builder (and is builder specific). A Compose implementation MUST support at least
> passing a path to a directory (absolute or realite) and an git repository URL, like [context](#context) does. Other context flavours MUST be previxed to avoid
> ambiguity with a `type://` prefix.

Reviewer: ```suggestion
When used as a list, the syntax should follow the `NAME=VALUE` format, where `VALUE` is a string. Validation beyond that is the responsibility of the image builder (and is builder specific). A Compose implementation MUST support
at least absolute and relative paths to a directory AND Git repository URLs, like [context](#context) does. Other context flavours MUST be prefixed to avoid
ambiguity with a `type://` prefix.
```
